### PR TITLE
Karambit

### PR DESCRIPTION
**Pull request by CptnCapitalism**
Adds a new melee weapon for the Syndicate, rebalanced based on the feedback I got.

## Why It's Good For The Game
While eswords are robust enough to kill an opponent in a few hits, their sound makes it obvious for anyone nearby that they have a traitor on the ship. This karambit deals less damage than an edagger, but gives agents the ability to "slit" a target's throat,causing heavy bleeding and making them unable to speak for a short time, allowing for a new way of silent murder.

## Changelog
:cl:
add: Added the karambit, a new Syndicate weapon which deals less damage than an edagger, but has a special attack that prevents your target from speaking for a short while.
add: Karambits can be found in Syndicate uplinks under Stealthy and Inconspicuous Weapons, and in the nuke ops lockers
/:cl:

### CODE REVIEW

**Comment by SouDescolado** [comment]
How broken would this be if you could insta-kill a sleeping target?

**Comment by CptnCapitalism** [comment]
> 
> 
> How broken would this be if you could insta-kill a sleeping target?

Not too bad, unless you combine it with a sleepy pen. But that feature could make sense on any sharp object.

**Comment by optimumtact** [comment]
Did you make a new PR?

also I still don't think we need more syndicate items

**Comment by CptnCapitalism** [comment]
Yes, this is a new PR (sorry, I'm new to using git). At least now I know how to push new commits to an existing one.

**Comment by CptnCapitalism** [comment]
Right now if a traitor wants to take out a target silently and doesn't want them to alert the station on comms, he could either buy a sleepypen + a poison kit, or a stechkin + a suppressor. Both of these options cost the traitor half his TCs, but more importantly, neither gives the victim a fighting chance. They are dead in a few seconds, and their body will either be spaced, or hidden in some random locker, effectively ending their round as a human.

This karambit allows the traitor to have a more "loud" loadout, with a chance for taking out single targets "quietly" when needed, while giving the victim a fair shot at fighting their attacker, since the weapon only does 6 damage in the silencing mode.

**Comment by Nickkoranda** [comment]
Why not replace the survival knife nukes have with the karambit instead? Lowkey a buff to the caravan ruin since the armory closet spawns in there.

**Comment by CptnCapitalism** [comment]
I already added them to the lockers at the nukie outpost where they start, but I could do that too. It deals less damage than the survival knife (12 instead of 15), but if someone wants to go for damage, they'll just use their esword anyway.

**Comment by Qustinnus** [comment]
@optimumtact traitor items are easy for new coders to make and can usually be unique/varied enough that they all fill their own niche so I'd say the more the merrier, we can just remove shitty traitor items if it becomes a problem

File: code/game/objects/items/kitchen.dm [hunk 1]
```diff
@@ -134,6 +134,37 @@
...
+	desc = "A karambit with a plasteel blade. Attacking an opponent's mouth will target their throat. Such an attack will cause heavy bleeding, which effectively mutes the target for a few seconds, but only deals half the normal damage due to the different technique."
+	force = 12
+	attack_verb = list("slashed", "sliced", "torn", "cut")
+	w_class = WEIGHT_CLASS_SMALL
+	sharpness = IS_SHARP
+
+
+/obj/item/kitchen/karambit/attack(mob/living/carbon/human/M, mob/living/carbon/human/user)
+	if(user.zone_selected == BODY_ZONE_PRECISE_MOUTH)
+		force = 6
```
Comment (nemvar) [inline code/game/objects/items/kitchen.dm]: force -= 6

File: code/game/objects/items/kitchen.dm [hunk 2]
```diff
@@ -134,6 +134,37 @@
...
+		to_chat(user, "<span class='userdanger'>You manage to slit [M]'s throat!</span>")
+		to_chat(M, "<span class='userdanger'>Your throat has been slit! You won't be able to speak for a while.</span>")
+		M.silent = 5
+		if(M.bleed_rate <= 4)
+			M.bleed_rate += 1.5
+			if(M.bleed_rate >= 4)
+				M.bleed_rate = 4
+		..()
+		force = 12
```
Comment (nemvar) [inline code/game/objects/items/kitchen.dm]: force += 6

**Comment by CptnCapitalism** [comment]
Good idea. I'll change it to "force /= 2" and "force *= 2", since the main idea is to halve the damage for the muting attack.

**Comment by nemvar** [comment]
Use *= 0.5

**Comment by optimumtact** [comment]
Floyd we have literally no way to adequately balance antag traitors anymore precisely because of that issue

**Comment by Qustinnus** [comment]
Mmh that sounds fair but I feel like there's still ways to get around that by introducing a system that circumvents the balance problems

**Comment by ExcessiveUseOfCobblestone** [comment]
Find someone who will merge this as-is or have it replace another uplink item, I'm not really interested in more traitor options that are just another way to directly kill a player.

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
above

**Comment by anconfuzedrock** [comment]
I dunno why you'd use this thing to kill another person since this is just a hatchet in power but the rad health scanner is rarely used and extremely unfun for the victim and unengaging for the user.

**Comment by ExcessiveUseOfCobblestone** [comment]
sounds like a good replacement item then.

**Pull request closed without merging**